docs/source/getting-started.md

```diff
@@ -44,6 +44,16 @@ iwr 'https://rover.apollo.dev/win/v0.17.2' | iex
 
 Rover is distributed on npm for integration with your JavaScript projects.
 
+#### Installation from NPM mirror
+
+If you are behind a proxy and install dependencies through a mirror or proxy of npm, add the following to your global or local `.npmrc`:
+
+```ini
+apollo_rover_download_host=https://your.mirror.com/repository
+```
+
+You can also set this value using the `APOLLO_ROVER_DOWNLOAD_HOST` environment var.
```

Comment:
```suggestion
You can also set this value using the `APOLLO_ROVER_DOWNLOAD_HOST` environment variable.
```

Reply:
looks like it's just a random error with installing npm on circleci cd server. Can you guys re-trigger that job and see if it resolves the check.

Reply:
Just re-triggered and it's all good. ✅

Before merging, would love to have an engineer take a closer look. (I am but a technical writer, hence the language suggestion. 😊)

@dbanty , would you be able to take a look while Avery is out?